src/AppBundle/Controller/Report/MoneyOutController.php

```diff
@@ -82,10 +82,8 @@ public function stepAction(Request $request, $reportId, $step, $transactionId =
         $form = $this->createForm(FormDir\Report\MoneyTransactionType::class, $transaction, [
             'step' => $step,
             'type'             => 'out',
-            'translator'       => $this->get('translator'),
```

Comment:
not used anymore

Reply:
Can you confirm that this is not used to translate the categories in the form? Also there are references to the translator within the configureOptions (            ->setAllowedTypes('translator', TranslatorInterface::class);
)

Reply:
the "translator" in that form is removed completely along with the "setAllowedTypes". 
I think we can consider deprecated the usage of the translator inside the form, as symfony (and our custom twig helpers) already do the job of translating once the key is given, and the translation_domain is set in the form